src/Web/WebMVC/libman.json

```diff
@@ -9,14 +9,6 @@
     {
       "provider": "unpkg",
       "library": "bootstrap@4.3.1",
-      "files": [
```

Comment:
These "files change" is included in another PR and don't seem necessary, for can't reproduce the error. See PR https://github.com/dotnet-architecture/eShopOnContainers/pull/994#issuecomment-479929262